Isfolkid/TextiFyrirSp2Isfolkid

```diff
@@ -0,0 +1,48 @@
+
+Þetta er gert í Therminal í Visual Studio Code.
+
+C:\Users\halld\Downloads\Háskóli_Íslands\sqlite3 data\isfolkid.db
```

Comment:
Þetta er ekki alveg nógu skýrt. Þetta þarf að vera relative path (svo þetta sé ekki bundið við *eina* ákveðna tölvu). Einnig er engin mappa sem heitir `data/` í þessu repo lengur, þannig þetta er out-of-date.